driver.cpp

```diff
@@ -32,10 +35,13 @@ namespace bitcoinfuzz
     {
         std::optional<std::vector<bool>> last_response{std::nullopt};
         for (auto& module : modules)
+        for (auto& module : modules)
         {
             std::optional<std::vector<bool>> res{module.second->deserialize_block(buffer)};
             if (!res.has_value() || res->empty()) continue;
             if (last_response.has_value()) assert(*res == *last_response);
+            if (!res.has_value() || res->empty()) continue;
```

Comment:
There are a lot of duplicated code. Should remove.

Reply:
Thanks for this. Lots of duplicate code came during rebase. I have fixed that!